### PR TITLE
Update malware detection documentation to include virustotal

### PR DESCRIPTION
**Pull request by guidomodarelli**
## Description

Update Malware detection documentation to include VirusTotal.

- [x] source/images/getting-started/malware-detection.png
- [x] source/images/wazuh-dashboard/navigating/endpoint-security.png (Malware Dection has a different description)
- [x] source/images/ansible/wazuh-dashboard-main.png (VirusTotal should not be present, old menu, Malware detection is not visible)
- [x] source/images/getting-started/endpoint-security-modules.png (different Malware Detection description)

Closes https://github.com/wazuh/wazuh-documentation/issues/7965

## Tests

- source/deployment-options/deploying-with-ansible/guide/install-indexer-dashboard.rst
- source/getting-started/components/wazuh-dashboard.rst
- source/getting-started/index.rst
- source/user-manual/wazuh-dashboard/navigating-the-wazuh-dashboard.rst

### Docs building
- [ ] Compiles without warnings.
### Code formatting and web optimization
- [ ] Uses three spaces indentation.
- [ ] Adds or updates meta descriptions accordingly.
- [ ] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).
### Writing style
- [ ] Uses present tense, active voice, and semi-formal registry.
- [ ] Uses short, simple sentences.
- [ ] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.

### CODE REVIEW

**Review by JuanGarriuz — APPROVED** [review]
LGTM!!

**Review by chantal-kelm — DISMISSED** [review]
LGTM!!

**Review by asteriscos — DISMISSED** [review]
LGTM :green_circle: 

image


image

image

File: source/images/getting-started/malware-detection.png
Comment (javimed) [inline source/images/getting-started/malware-detection.png:1]: Are we ok displaying sample data rather than actual data?

**Review by javimed — APPROVED** [review]

File: source/images/getting-started/malware-detection.png
Comment (guidomodarelli) [reply source/images/getting-started/malware-detection.png:1]: I can delete the warning

**Review by Desvelao — APPROVED** [review]
LGTM!

**Review by javimed — APPROVED** [review]

**Pull request merged**